### ISSUE TITLE
shopify help connect shows the same content twice

### ISSUE THREAD

**Issue by attaboy** [Shopify/shopify-app-cli#549]
Typing `shopify help connect` causes (almost) the same help output to appear twice:

```
$ shopify help connect
Connect a Shopify-App-Cli project. Restores the ENV file
  Usage: shopify connect

Connect a Shopify-App-Cli project. Restores the Env file
Usage: shopify connect
```

**Comment by attaboy** [comment]
@mkevinosullivan, I'm not sure if #546 may address this already?

**Comment by mkevinosullivan** [comment]
Yes it will!

**Comment by mkevinosullivan** [comment]
Fixed in #546
